### ISSUE TITLE
Error when issuing conda.exe update --all

### ISSUE THREAD

**Issue by liubenyuan** [conda/conda#1764]
```
PS C:\cem> conda.exe update --all
Fetching package metadata: ..
Solving package specifications: ............................................................................................................................................................................................................................................................................
Error: Unsatisfiable package specifications.
Generating hint:
[      COMPLETE      ]|##################################################| 100%


Hint: 'hdf5' has unsatisfiable dependencies (see 'conda info hdf5')

Note that the following features are enabled:
  - vc10
  - vc14
  - vc9
```

and

```
PS C:\cem> conda.exe info hdf5
Fetching package metadata: ..

hdf5 1.8.9 0
------------
file name   : hdf5-1.8.9-0.tar.bz2
name        : hdf5
version     : 1.8.9
build number: 0
build string: 0
channel     : http://repo.continuum.io/pkgs/free/win-64/
size        : 2.5 MB
date        : 2014-08-22
license     : BSD-like (http://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING)
license_family: BSD
md5         : 8d00b379ea8c7fac7abb6168216b13a8
org_name    : hdf5-1.8.15
installed environments:
dependencies:

hdf5 1.8.13 0
-------------
file name   : hdf5-1.8.13-0.tar.bz2
name        : hdf5
version     : 1.8.13
build number: 0
build string: 0
channel     : http://repo.continuum.io/pkgs/free/win-64/
size        : 2.3 MB
date        : 2014-09-11
license     : BSD-like (http://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING)
license_family: BSD
md5         : 14fececf963b59dd7f1ba4045957a7c7
org_name    : hdf5-1.8.15
installed environments:
dependencies:

hdf5 1.8.15.1 2
---------------
file name   : hdf5-1.8.15.1-2.tar.bz2
name        : hdf5
version     : 1.8.15.1
build number: 2
build string: 2
channel     : http://repo.continuum.io/pkgs/free/win-64/
size        : 1.4 MB
date        : 2015-10-27
license     : BSD-like (http://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING)
license_family: BSD
md5         : da9a90f534cf9e47b03156af5ddadd7d
org_name    : hdf5-1.8.15
installed environments:
dependencies:
    zlib 1.2*

hdf5 1.8.15.1 vc9_3
-------------------
file name   : hdf5-1.8.15.1-vc9_3.tar.bz2
name        : hdf5
version     : 1.8.15.1
build number: 3
build string: vc9_3
channel     : http://repo.continuum.io/pkgs/free/win-64/
size        : 1.5 MB
date        : 2015-10-14
features    : vc9
license     : BSD-like (http://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING)
license_family: BSD
md5         : 15277b51954dcb35e3e03aa87efb9f28
org_name    : hdf5-1.8.15
installed environments:
dependencies:
    zlib

hdf5 1.8.15.1 vc14_3
--------------------
file name   : hdf5-1.8.15.1-vc14_3.tar.bz2
name        : hdf5
version     : 1.8.15.1
build number: 3
build string: vc14_3
channel     : http://repo.continuum.io/pkgs/free/win-64/
size        : 1.6 MB
date        : 2015-10-28
features    : vc14
license     : BSD-like (http://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING)
license_family: BSD
md5         : 85eeb204341c34010d347cdbc67d2974
org_name    : hdf5-1.8.15
installed environments:
dependencies:
    zlib

hdf5 1.8.15.1 vc10_3
--------------------
file name   : hdf5-1.8.15.1-vc10_3.tar.bz2
name        : hdf5
version     : 1.8.15.1
build number: 3
build string: vc10_3
channel     : http://repo.continuum.io/pkgs/free/win-64/
size        : 1.4 MB
date        : 2015-10-14
features    : vc10
license     : BSD-like (http://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING)
license_family: BSD
md5         : e33594a7a8871a564b3cdf49db1d4675
org_name    : hdf5-1.8.15
installed environments:
    C:\Anaconda3
dependencies:
    zlib
```

**Comment by groutr** [comment]
This is rather unfortunate.  What version of python is installed in the environment?  Could you please post the output of `conda list`?

**Comment by liubenyuan** [comment]
```
# packages in environment at C:\Anaconda3:
#
_license                  1.1                      py34_0
alabaster                 0.7.6                    py34_0
anaconda                  2.3.0                np19py34_0
anaconda-client           1.0.2                    py34_0
argcomplete               1.0.0                    py34_0
astropy                   1.0.4                np19py34_0
babel                     2.1.1                    py34_0
bcolz                     0.11.3                   py34_0
beautiful-soup            4.3.2                    py34_1
beautifulsoup4            4.3.2                     <pip>
binstar                   0.12                          2
bitarray                  0.8.1                    py34_1
blaze                     0.8.3                     <pip>
blaze-core                0.8.3                    py34_0
blz                       0.6.2                np19py34_1
bokeh                     0.10.0                   py34_0
boto                      2.38.0                   py34_0
bottleneck                1.0.0                np19py34_0
certifi                   14.05.14                 py34_0
cffi                      1.2.1                    py34_0
clyent                    0.4.0                    py34_0
colorama                  0.3.3                    py34_0
comtypes                  1.1.2                    py34_0
conda                     3.18.3                   py34_0
conda-build               1.17.0                   py34_0
conda-env                 2.4.4                    py34_0
configobj                 5.0.6                    py34_0
cryptography              0.9.3                    py34_0
cython                    0.23.4                   py34_0
cytoolz                   0.7.3                    py34_0
datashape                 0.4.7                    py34_0
decorator                 4.0.4                    py34_0
docutils                  0.12                     py34_1
dynd-python               0.6.5                np19py34_0
fastcache                 1.0.2                    py34_0
flask                     0.10.1                   py34_1
funcsigs                  0.4                      py34_0
greenlet                  0.4.9                    py34_0
h5py                      2.5.0                np19py34_2
hdf5                      1.8.15.1                 vc10_3  [vc10]
idna                      2.0                      py34_0
ipykernel                 4.1.0                    py34_0
ipython                   4.0.0                    py34_0
ipython-genutils          0.1.0                     <pip>
ipython-notebook          4.0.4                    py34_0
ipython-qtconsole         4.0.1                    py34_0
ipython_genutils          0.1.0                    py34_0
itsdangerous              0.24                     py34_0
jdcal                     1.0                      py34_0
jedi                      0.9.0                    py34_0
jinja2                    2.8                      py34_0
jsonschema                2.4.0                    py34_0
jupyter-client            4.1.1                     <pip>
jupyter-core              4.0.6                     <pip>
jupyter_client            4.1.1                    py34_0
jupyter_core              4.0.6                    py34_0
launcher                  1.0.0                         1
llvmlite                  0.7.0                    py34_5
lxml                      3.4.4                    py34_0
markupsafe                0.23                     py34_0
matplotlib                1.4.3                np19py34_1
menuinst                  1.0.4                    py34_0
mistune                   0.7.1                    py34_0
mock                      1.3.0                    py34_0
msvc_runtime              1.0.0                    vc10_0  [vc10]
multipledispatch          0.4.8                    py34_0
nbconvert                 4.0.0                    py34_0
nbformat                  4.0.1                    py34_0
networkx                  1.10                     py34_0
nltk                      3.0.5                    py34_0
node-webkit               0.10.1                        0
nose                      1.3.7                    py34_0
notebook                  4.0.5                    py34_0
numba                     0.21.0               np19py34_0
numexpr                   2.4.4                np19py34_0
numpy                     1.9.3                    py34_0
odo                       0.3.4                    py34_0
openpyxl                  2.0.2                    py34_0
pandas                    0.16.2               np19py34_0
patch                     2.5.9                         0
path.py                   8.1.2                    py34_0
patsy                     0.4.0                np19py34_0
pbr                       1.3.0                    py34_0
pep8                      1.6.2                    py34_0
pickleshare               0.5                      py34_0
pillow                    3.0.0                    py34_0
pip                       7.1.2                    py34_0
ply                       3.8                      py34_0
psutil                    3.2.2                    py34_0
py                        1.4.30                   py34_0
pyasn1                    0.1.7                    py34_0
pycosat                   0.6.1                    py34_0
pycparser                 2.14                     py34_0
pycrypto                  2.6.1                    py34_3
pyflakes                  1.0.0                    py34_0
pygments                  2.0.2                    py34_0
pyopenssl                 0.15.1                   py34_1
pyparsing                 2.0.3                    py34_0
pyqt                      4.10.4                   py34_1
pyreadline                2.1                      py34_0
pytables                  3.2.2                np19py34_0
pytest                    2.8.1                    py34_0
python                    3.4.3                         3
python-dateutil           2.4.2                    py34_0
pytz                      2015.6                   py34_0
pywin32                   219                      py34_0
pyyaml                    3.11                     py34_2
pyzmq                     14.7.0                   py34_0
qtconsole                 4.1.0                    py34_0
requests                  2.8.1                    py34_0
rope                      0.9.4                    py34_1
rope-py3k-0.9.4           1                         <pip>
runipy                    0.1.3                    py34_0
scikit-image              0.11.3               np19py34_0
scikit-learn              0.16.1               np19py34_0
scipy                     0.16.0               np19py34_0
setuptools                18.4                     py34_0
simplegeneric             0.8.1                    py34_0
six                       1.10.0                   py34_0
snowballstemmer           1.2.0                    py34_0
sockjs-tornado            1.0.1                    py34_0
sphinx                    1.3.1                    py34_0
sphinx-rtd-theme          0.1.7                     <pip>
sphinx_rtd_theme          0.1.7                    py34_0
spyder                    2.3.7                    py34_1
spyder-app                2.3.7                    py34_0
sqlalchemy                1.0.8                    py34_0
statsmodels               0.6.1                np19py34_0
sympy                     0.7.6.1                  py34_0
tables                    3.2.2                     <pip>
toolz                     0.7.4                    py34_0
tornado                   4.2.1                    py34_0
traitlets                 4.0.0                    py34_0
ujson                     1.33                     py34_0
werkzeug                  0.10.4                   py34_0
wheel                     0.26.0                   py34_1
xlrd                      0.9.4                    py34_0
xlsxwriter                0.7.6                    py34_0
xlwings                   0.4.1                    py34_0
xlwt                      1.0.0                    py34_0
zlib                      1.2.8                    vc10_2  [vc10]
```

**Comment by coobas** [comment]
I experience the same. The reason might be in qt/libpng dependencies and vc versions:
```
C:\Users\jakub\Desktop>conda update --all --alt-hint
Fetching package metadata: ....
Solving package specifications: .........................................................................................................................................................................................................................................
Error: Unsatisfiable package specifications.
Generating minimal hint:
[      COMPLETE      ]|##################################################| 100%

The following set of clauses is unsatisfiable:

libpng-1.6.17-vc14_1 => not libpng-1.6.17-vc9_1
qt-4.8.7-vc10_4 => libpng-1.6.17-vc9_1
qt-4.8.7-vc10_4
libpng-1.6.17-vc14_1
```

Btw, this is a fresh Anaconda3-2.4.0-Windows-x86_64 install on Win8.1.

**Comment by chadkennedyonline** [comment]
Had same issue. This seems to be the core of the problem: #1779. Don't know of a fix yet.

**Comment by kalefranz** [comment]
This should now be resolved with a combination of package fixes and solver fixes in conda 4.0.
